src/components/Guides.tsx

```diff
@@ -279,6 +279,35 @@ const guides: any = {
 		],
 	}),
 
+	'raspberrypi3-64': (
+		sdk: BalenaSdk.BalenaSDK,
+		app?: BalenaSdk.Application,
+		deviceType?: BalenaSdk.DeviceType,
+	) => ({
+		intro: (
+			<>
+				{renderGenericInfo(deviceType?.name)}
+
+				<Txt.p mt={3}>
+					<Alert info>
+						Occasionally you may see a message like "Rosetta needs 1907.35 MB
+						RAM but only 966.82 MB is available for use.", this simply means
+						there are no tasks present in the queue at that time - keep your
+						device online and more will be added soon! Read more in{' '}
+						<Link href="#faqs">our FAQ</Link>.
```

Comment:
Let's use './Link' instead of rendition `Link`, so that the page doesn't reload on link clicks.